Assets/Scripts/Cards/Card.cs

```diff
@@ -16,6 +16,8 @@ public abstract class Card : ScriptableObject {
 	public int attackPower;
 	public int health;
 
+	public bool isActive;
```

Comment:
Is this set to true when the card is on the battlefield?

Reply:
Moreover, does this conflict with any Unity properties? Let's consider a more precise name.